tracing/ops_tracing/const.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2025 Canonical Ltd.
```

Comment:
I think it's generally better if things belong where they're needed / grouped rather than in a central "const" or "utils" file. So `EXPORT_TIMEOUT` would go in `_export.py`, and `BUFFER_FILE` in `_buffer.py`, and `Config` where that belongs. In addition `Config` is a type/class, not a const.